src/api/public/apidocs-new/OBS-v2.10.50.yaml

```
@@ -69,10 +69,13 @@ paths:
 
   /person:
     $ref: 'paths/person.yaml'
-  /person/register:
-    $ref: 'paths/person_register.yaml'
+  /person/{login}:
+    $ref: 'paths/person_login.yaml'
   /person/{login}/group:
     $ref: 'paths/person_login_group.yaml'
+  /person/register:
+    $ref: 'paths/person_register.yaml'
```

```suggestion
  /person/register:
    $ref: 'paths/person_register.yaml'
  /person/{login}:
    $ref: 'paths/person_login.yaml'
  /person/{login}/group:
    $ref: 'paths/person_login_group.yaml'
```

Nitpick: I would list before fixed routes, and then routes with parameters. What do you think?

---

I was going for alphabetical ordering, but I like your idea